client/components/jetpack/threat-list-header/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { CompactCard } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+const ThreatListHeader = () => {
+	const translate = useTranslate();
+
+	return (
+		<CompactCard className="threat-list-header threat-list-header__wrapper">
+			<div className="threat-list-header__title title__severity">{ translate( 'Severity' ) }</div>
+			<div className="threat-list-header__title title__details">{ translate( 'Details' ) }</div>
+			<div className="threat-list-header__title title__autofix">{ translate( 'Auto Fix' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Auto fix' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/more-informative-scan/threat-list)